src/plugins/manualRowResize/test/manualRowResize.e2e.js

```diff
@@ -133,6 +133,43 @@ describe('manualRowResize', () => {
     expect(rowHeight(spec().$container, 2)).toEqual(defaultRowHeight + 1);
   });
 
+  it('should trigger beforeRowResize event after row height changes', () => {
+    const beforeRowResizeCallback = jasmine.createSpy('beforeRowResizeCallback');
+
+    handsontable({
+      data: Handsontable.helper.createSpreadsheetData(5, 5),
+      rowHeaders: true,
+      manualRowResize: true,
+      beforeRowResize: beforeRowResizeCallback
+    });
+
+    expect(rowHeight(spec().$container, 0)).toEqual(defaultRowHeight + 2);
+
+    resizeRow(0, 100);
+    expect(beforeRowResizeCallback).toHaveBeenCalledWith(100, 0, false, void 0, void 0, void 0);
+    expect(rowHeight(spec().$container, 0)).toEqual(101);
+  });
+
+  it('should appropriate resize rowHeight after beforeRowResize call a few times', () => {
+    const hot = handsontable({
+      data: Handsontable.helper.createSpreadsheetData(3, 3),
+      rowHeaders: true,
+      manualRowResize: true
+    });
+
+    expect(rowHeight(spec().$container, 0)).toEqual(24);
+
+    hot.addHook('beforeRowResize', () => 100);
+
+    hot.addHook('beforeRowResize', () => 200);
+
+    hot.addHook('beforeRowResize', () => {});
```

Comment:
Accordingly to typings in d.ts file, `beforeRowResize ` should return `number|void`.